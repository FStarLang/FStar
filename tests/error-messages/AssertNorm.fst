module AssertNorm

(* The failure is in the assert_norm that's what should be highlighted. *)

[@@expect_failure [19]]
let f (x y z : int) =
  assert_norm (x + x == y + y);
  assert (x + y == y + x);
  42
