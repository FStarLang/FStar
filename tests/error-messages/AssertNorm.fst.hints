[ "›\rQS¬Ó¤\u007f[ŽJoñ¼ä¬", [] ]