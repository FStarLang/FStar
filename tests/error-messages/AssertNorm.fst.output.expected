* Info at AssertNorm.fst(7,2-7,13):
  - Expected failure:
  - Assertion failed
  - The SMT solver could not prove the query. Use --query_stats for more
    details.
  - See also AssertNorm.fst(7,14-7,30)

